Makefile

```diff
@@ -58,7 +74,23 @@ ifeq ($(BUILDARCH),amd64)
 endif
 
 # Makefile configuration options
-CONTAINER_NAME=calico/kube-controllers
+BUILD_IMAGE?=calico/kube-controllers
+PUSH_IMAGES?=$(BUILD_IMAGE) quay.io/calico/kube-controllers
+RELEASE_IMAGES?=gcr.io/projectcalico-org/kube-controllers eu.gcr.io/projectcalico-org/kube-controllers asia.gcr.io/projectcalico-org/kube-controllers us.gcr.io/projectcalico-org/kube-controllers
```

Comment:
RELEASE_IMAGES should be blank for this repo, the old Makefile didn't push images to GCR

Reply:
Updated. Back at you.